201-servicebus-private-endpoint/azuredeploy.json

```diff
@@ -0,0 +1,1040 @@
+      "dependsOn": [
+        "[variables('blobPrivateDnsZoneId')]",
+        "[variables('vnetId')]"
+      ],
+      "properties": {
+        "registrationEnabled": false,
+        "virtualNetwork": {
+          "id": "[variables('vnetId')]"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Network/privateEndpoints",
+      "apiVersion": "2020-04-01",
+      "name": "[parameters('serviceBusNamespacePrivateEndpointName')]",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[variables('vnetId')]",
+        "[variables('serviceBusNamespaceId')]"
+      ],
+      "properties": {
+        "privateLinkServiceConnections": [
+          {
+            "name": "[parameters('serviceBusNamespacePrivateEndpointName')]",
+            "properties": {
+              "privateLinkServiceId": "[variables('serviceBusNamespaceId')]",
+              "groupIds": [
+                "[variables('serviceBusNamespacePrivateEndpointSubResource')]"
+              ]
+            }
+          }
+        ],
+        "subnet": {
+          "id": "[variables('subnetId')]"
+        },
+        "customDnsConfigs": [
+          {
+            "fqdn": "[concat(parameters('serviceBusNamespaceName'), variables('serviceBusPublicDNSZoneForwarder'))]"
+          }
+        ]
+      }
+    },
+    {
+      "type": "Microsoft.Network/privateEndpoints",
+      "apiVersion": "2020-04-01",
+      "name": "[parameters('blobStorageAccountPrivateEndpointName')]",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[variables('vnetId')]",
+        "[variables('blobStorageAccountId')]"
+      ],
+      "properties": {
+        "privateLinkServiceConnections": [
+          {
+            "name": "[parameters('blobStorageAccountPrivateEndpointName')]",
+            "properties": {
+              "privateLinkServiceId": "[variables('blobStorageAccountId')]",
+              "groupIds": [
+                "[variables('blobStorageAccountPrivateEndpointSubResource')]"
+              ]
+            }
+          }
+        ],
+        "subnet": {
+          "id": "[variables('subnetId')]"
+        },
+        "customDnsConfigs": [
+          {
+            "fqdn": "[concat(parameters('blobStorageAccountName'), variables('blobPublicDNSZoneForwarder'))]"
+          }
+        ]
+      }
+    }
+  ],
+  "outputs": {
+    "serviceBusNamespacePrivateEndpoint": {
+      "value": "[reference(resourceId('Microsoft.Network/privateEndpoints', parameters('serviceBusNamespacePrivateEndpointName')), '2020-04-01', 'Full')]",
+      "type": "object"
+    },
+    "blobStorageAccountPrivateEndpoint": {
+      "value": "[reference(resourceId('Microsoft.Network/privateEndpoints', parameters('blobStorageAccountPrivateEndpointName')), '2020-04-01', 'Full')]",
+      "type": "object"
+    },
+    "serviceBusNamespace": {
+      "value": "[reference(resourceId('Microsoft.ServiceBus/namespaces', parameters('serviceBusNamespaceName')), '2017-04-01', 'Full')]",
```

Comment:
update apiVersion

Reply:
I'm using the latest apiVersion for Service Bus: 2018-01-01-preview